### PR TITLE
Error audit protocols

### PR DESCRIPTION
**Pull request by sklump**

### CODE REVIEW

File: aries_cloudagent/messaging/jsonld/routes.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ async def sign(request: web.BaseRequest):
     try:
         context = request.app["request_context"]
         wallet: BaseWallet = await context.inject(BaseWallet)
+        wallet: BaseWallet = await context.inject(BaseWallet)
```
Comment (andrewwhitehead) [inline aries_cloudagent/messaging/jsonld/routes.py:43]: Accidental commit?

File: aries_cloudagent/protocols/connections/v1_0/routes.py [hunk 2]
```diff
@@ -373,16 +387,21 @@ async def connections_accept_invitation(request: web.BaseRequest):
...
+        my_endpoint = request.query.get("my_endpoint") or None
+        request = await connection_mgr.create_request(connection, my_label, my_endpoint)
+        result = connection.serialize()
     except StorageNotFoundError as err:
         raise web.HTTPNotFound(reason=err.roll_up) from err
-    connection_mgr = ConnectionManager(context)
-    my_label = request.query.get("my_label") or None
-    my_endpoint = request.query.get("my_endpoint") or None
-    request = await connection_mgr.create_request(connection, my_label, my_endpoint)
+    except (StorageError, WalletError, ConnectionManagerError, BaseModelError) as err:
```
Comment (andrewwhitehead) [inline aries_cloudagent/protocols/connections/v1_0/routes.py:400]: Should it just catch BaseError?

File: aries_cloudagent/messaging/jsonld/routes.py (hunk 1 above)
Comment (sklump) [reply aries_cloudagent/messaging/jsonld/routes.py:43]: Good catch: I will fix it

File: aries_cloudagent/protocols/connections/v1_0/routes.py (hunk 2 above)
Comment (sklump) [reply aries_cloudagent/protocols/connections/v1_0/routes.py:400]: I think it's OK to be specific here: catching an omnibus error will probably make LGTM-bot squawk.

Comment (sklump) [reply aries_cloudagent/protocols/connections/v1_0/routes.py:400]: I'd prefer to see what can happen in particular.

**Pull request merged**